src/libs/httpserver/indexDataFetcher.ts

```diff
@@ -1,12 +1,27 @@
-import { replaceAll } from '../functions/stringHelpers'
-import { fetchCozyDataForSlug } from '../client'
+import CozyClient from 'cozy-client'
+
+import { AppData, AppAttributes, CozyData } from '/libs/httpserver/models'
+import { fetchCozyDataForSlug } from '/libs/client'
 import { getCookie } from './httpCookieManager'
 import { logToSentry } from '/libs/Sentry'
+import { replaceAll } from '/libs/functions/stringHelpers'
+
+interface FetchAppDataForSlugResult {
+  cookie: string
+  templateValues: {
+    [P in keyof Partial<Omit<AppAttributes, 'CozyData'>>]: string
+  } & {
+    CozyData: string
+  }
+}
```

Comment:
This type seems from a mix between an old type organisation and a new one.

Here you Omit `CozyData` from `AppAttributes` but `AppAttributes` does not contain this attribute
And then you add `CozyData` to the final type again. This seems to cancel the Omitted one.

Reply:
Fixed but CozyData isn't in AppAttributes though